service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/dataproccluster/startup.sh

```diff
@@ -0,0 +1,768 @@
+#!/bin/bash
+#
+# Name: startup.sh
+#
+# Description
+#   Initialization script to setup Terra configurations in a Dataproc cluster manager node
+#
+# Execution details
+#   This script is executed as root on all Dataproc nodes.
+#
+#   Note that the guest attribute is set to DONE whether the script runs successfully or not.
```

Comment:
Please list the specific guest attribute(s) and metadata used for startup orchestration. That would be super helpful to have in one place.

- attributes/dataproc-role: read by this script to determine if this is the dataproc head node or a worker node

- instance/guest-attributes/startup_script/status: set by this script to indicate STARTUP/COMPLETE/ERROR
- instance/guest-attributes/startup_script/message: set by this script with more information on any ERROR status

- instance/attributes/terra-cli-server: read by this script to configure the terra CLI server
- instance/attributes/terra-workspace-id: read by this script to configure the terra CLI workspace

Reply:
Thanks for enumerating these out!